File: Modul 5/c/PRAK504-2310817110007-MuhammadDaffaMusyafa.c
#include <stdio.h>
int reverse(int n){
// Lengkapi Function ini
    int isi = 0;
    while (n != 0)
    {
        isi = isi*10;
        isi = isi+n%10;
        n = n/10;}
        return isi;}
int main() {
 int A, B;
 scanf("%d %d",&A,&B);
 A=reverse(A);
 B=reverse(B);
 int C = A+B;
 printf("%d",reverse(C));
}

File: Modul 5/py/PRAK504-2310817110007-MuhammadDaffaMusyafa.py
def reverse(n):
    isi = 0
    while n != 0:
        isi = (isi * 10)+(n%10)
        n = n//10
    return isi

A, B = map(int, input().split())

A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))